daemon/cmd/cleanup.go

```diff
@@ -61,46 +65,9 @@ func (c *cleanupFuncList) Run() {
 	}
 }
 
-func (d *daemonCleanup) registerSigHandler() <-chan struct{} {
```

Comment:
Where did we have the signal handlers?

Reply:
We're now relying on uber/fx's signal handling (via `app.Run()`).

Reply:
https://github.com/uber-go/fx/blob/4f51a19264fc8d0bf7b87ef7a8855feb96d264b7/app.go#L716